assets/src/edit-story/elements/media/editPanMoveable.js

```diff
@@ -47,21 +49,54 @@ function EditPanMoveable({
   const translateRef = useRef([0, 0]);
   const transformFlip = getTransformFlip(flip);
 
-  const update = () => {
+  const update = useCallback(() => {
     const [tx, ty] = translateRef.current;
     fullMedia.style.transform = `translate(${tx}px, ${ty}px) ${
       transformFlip ?? ''
     }`;
     croppedMedia.style.transform = `translate(${tx}px, ${ty}px) ${
       transformFlip ?? ''
     }`;
-  };
+  }, [croppedMedia, fullMedia, transformFlip]);
 
   // Refresh moveables to ensure that the selection rect is always correct.
   useEffect(() => {
     moveableRef.current.updateRect();
   });
 
+  useKeyDownEffect(
+    { current: croppedMedia },
+    { key: ['up', 'down', 'left', 'right'], shift: true },
+    ({ key, shiftKey }) => {
+      let { dx, dy } = getKeyboardMovement(key, shiftKey);
+      if (flip?.vertical) {
+        dy = -dy;
+      }
+      if (flip?.horizontal) {
+        dx = -dx;
+      }
+      const panFocalX = getFocalFromOffset(width, mediaWidth, offsetX - dx);
+      const panFocalY = getFocalFromOffset(height, mediaHeight, offsetY - dy);
+      setProperties({
```

Comment:
It seems to lag currently when not clicking the keys slowly -- would be good to create a follow-up issue for looking into that.

Reply:
Yeah, there's a lot of lag currently. It's beginning to be a problem :'(